daily_notebook/20240701__modelling_bat_source_pressure_respect_to_microphone.ipynb

```diff
@@ -0,0 +1,727 @@
+{
```

Comment:
<p>We should do both of the frequency groups?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/uw-echospace/duty-cycle-investigation/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>